src/components/navigation-header/components/primary-nav-link/primary-nav-link.module.css

```diff
@@ -1,22 +1,24 @@
 .root {
-  /* Composition */
-  composes: g-focus-ring-from-box-shadow-dark from global;
+	/* Composition */
+	composes: g-focus-ring-from-box-shadow-dark from global;
 
-  /* CSS properties */
-  border-radius: 5px;
-  color: var(--token-color-palette-neutral-400);
-  display: block;
-  padding-bottom: 8px;
-  padding-left: var(--header-dropdown-menu-padding-left-right);
-  padding-right: var(--header-dropdown-menu-padding-left-right);
-  padding-top: 8px;
+	/* CSS properties */
+	align-items: center;
+	gap: 6px;
+	border-radius: 5px;
+	color: var(--token-color-palette-neutral-400);
+	display: flex;
```

Comment:
Meaningful changes here, to support the optional trailing icon:
- `display: flex`
- `gap: 6px`
- `align-items: center`
The rest is formatting.

Reply:
Diff without whitespace: https://github.com/hashicorp/dev-portal/pull/996/files?w=1